src/extension/src/__main__.py

```diff
@@ -49,7 +49,7 @@ def main(argv):
         json_file_handler = JsonFileHandler(logger)
         ext_env_handler = ExtEnvHandler(json_file_handler)
         env_layer = EnvLayer()
-        env_manager = EnvManager(env_layer)
+        env_manager = EnvHealthManager(env_layer)
```

Comment:
env_health_manager -- consistency for ease of maintainability.

Reply:
Resolved